src/app/Inventory/Item.tsx

```diff
@@ -115,20 +117,31 @@ const Item: React.FC<Props> = ({ item }) => {
             style={inlineStyles}
           />
         </div>
-        <Input
-          value={product_name || ""}
-          placeholder="product name"
-          onChange={(v) => update("product_name", v)}
-          onBlur={() => handleSave("product_name")}
-          style={inlineStyles}
-        />
+        <div>
+          <Input
+            value={product_name || ""}
+            placeholder="product name"
+            onChange={(v) => update("product_name", v)}
+            onBlur={() => handleSave("product_name")}
+            style={inlineStyles}
+          />
+        </div>
+        <div>
+          <Input
+            value={manufacturer || ""}
+            placeholder="manufacturer"
+            onChange={(v) => update("manufacturer", v)}
+            onBlur={() => handleSave("manufacturer")}
+            style={inlineStyles}
+          />
+        </div>
         <div>
           <SelectCreatable
             options={categoryOptions(categories)}
             value={categoryValue}
             onChange={(option: Option<number>) => {
-              update("categoryId", option.value)
-              updateCategory(option.value)
+              update("categoryId", option.value);
+              updateCategory(option.value);
```

Comment:
Your IDE may be adding semicolons. Not a blocker for this PR, but I typically remove these.